Packs/Jira/Integrations/JiraV2/JiraV2.yml

```
@@ -553,7 +553,21 @@ script:
       name: role_name
     description: Adds the user to a project.
     name: jira-add-user-to-project
-  - arguments: []
+  - arguments: 
+    - description: The ID of the service desk from which the organization list will be returned. This can alternatively be a project identifier.
+      name: projectKey
+      required: false
+    - description: The starting index of the returned objects. Base index 0. See the Pagination section for more details.
```

```suggestion
    - description: The starting index of the returned objects. Base index 0. See the Pagination section for more details.
```
Where is the Pagination section?